src/Telemetry/Logging/HttpClientLogging/Program.cs

```diff
@@ -42,13 +40,21 @@
 // We add the logging to the named HttpClient and configure it via action:
 httpClientBuilder.AddExtendedHttpClientLogging(options =>
 {
-    options.RouteParameterDataClasses.Add("userId", DataTaxonomy.PrivateData);
+    // In Formatted mode all request path parameters aren't logged as separate tags:
```

Comment:
I'd personally like to rephrase this. In my opinion, it is difficult to read. Something like this:

```suggestion
    // In Formatted mode all request path parameters are logged as part of HTTP URL path.
    // If you use Structured mode they will be logged as separate tags and the HTTP URL will contain the HTTP request route.
```

Reply:
Applied, thanks!